projects/step-frontend/src/lib/modules/execution/components/alt-execution-report/alt-execution-report.component.ts

```diff
@@ -23,6 +24,8 @@ import { map } from 'rxjs';
   encapsulation: ViewEncapsulation.None,
 })
 export class AltExecutionReportComponent {
+  @ViewChild(GridsterComponent) gridster?: GridsterComponent;
```

Comment:
Is this field used?
Please remove it, in case if it's not required.
Otherwise use `viewChild` signal function, instead of decorator.